backend/src/v5/utils/helper/yup.js

```diff
@@ -26,9 +26,9 @@ const YupHelper = { validators: {}, types: { strings: {} }, utils: {} };
 
 YupHelper.utils.stripWhen = (schema, cond) => Yup.lazy((value) => (cond(value) ? schema.strip() : schema));
 
-YupHelper.validators.alphanumeric = (yupObj) => yupObj.matches(/^[\w|_|-]*$/,
+YupHelper.validators.alphanumeric = (yupObj) => yupObj.matches(/^[\w|_|.|-]*$/,
```

Comment:
alphanumeric is used in other places, we can't just add fullstop here, use a boolean